docs/specs/phase 3c/02-cart-system/tasks.md

```
@@ -0,0 +1,239 @@
+
+- `[P]` Parallelizable - can run concurrently with other [P] tasks
+- `[S]` Sequential - depends on previous tasks
+- `[ ]` Not started
+- `[X]` Completed
+- `[~]` In progress
+
+---
+
+## Phase 1: Infrastructure
+
+### 1.1 shadcn Setup
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)